frontend/src/hooks/useFields.ts

```diff
@@ -108,6 +109,7 @@ export function useFields<T extends Zaak = Zaak>(
     {
       name: "einddatum",
       type: "daterange",
+      filterLookup: "startdatum__gte,startdatum__lte",
```

Comment:
Same issue as the other field, but also check the actual field, this is not the start but end date filter here...